__tests__/src/actions/contacts.js

```diff
@@ -0,0 +1,44 @@
+import {
+  START_CONTACTS_FETCH,
+  CONTACTS_UPDATED,
+  ADD_CONTACT,
+  startContactsFetch,
+  contactsUpdated,
+  addContact,
+} from '../../../src/actions/contacts';
+
+describe('contacts action creators', () => {
+  test('startContactsFetch', () => {
+    expect(startContactsFetch()).toEqual({
+      type: START_CONTACTS_FETCH,
+    });
+  });
+
+  test('contactsUpdated', () => {
+    const mockAddress = '0xtestaddress';
```

Comment:
There is no need to extract address into separate declaration, you can just inline it in profileMock.

Reply:
Done